static/js/store/nodes.js

```diff
@@ -493,6 +498,17 @@ var app = app || {};
         }
     }
 
+    function updateGroupRouteVisibility(event) {
+      console.log(event)
+      if (event.data.isVisible) {
+        nodes[event.id].data.hiddenRoutes.splice(nodes[event.id].data.hiddenRoutes.indexOf(event.data.route.id), 1);
```

Comment:
for this and below: how come? I don't think the hiddenRoutes array stops a route being assigned, it just stops it from rendering. Do I need something like `hideRoutesAscending` so that parent groups inherit the hiddenRoutes of its children?